package.json

```diff
@@ -26,13 +26,13 @@
     "del": "^2.2.0",
     "gulp": "^3.9.1",
     "gulp-autoprefixer": "^3.1.0",
-    "gulp-sass": "^2.2.0",
+    "gulp-sass": "^4.0.0",
     "gulp-sass-glob": "^1.0.0",
     "gulp-sourcemaps": "^1.6.0",
-    "gulp-stylelint": "^2.0.0",
+    "gulp-stylelint": "^2.0.2",
     "gulp-uglify": "^1.5.3",
     "highlight.js": "^9.1.0",
-    "jquery": "^2.2.1",
+    "jquery": "^3.0.0",
```

Comment:
I'm afraid this can leads to unexpected breaking changes and I don't see how it is related to the project setup to work again. Can you please explain?

Reply:
It's not, but it eliminates a high-value audit risk. I haven't noticed any breaking changes so far--if I remember correctly proper use of version 2 should be forward-compatible. 

But it's not an essential change here.

Reply:
Since Mandelbrot can be customized in many ways, people might have used jQuery in a way that could break with this change. I would rather leave it aside for now except if plan to make a major release.

Reply:
OK, reverted the non-essential local dev changes.